addons/project/models/project_task.py

```diff
@@ -1608,3 +1608,10 @@ def get_unusual_days(self, date_from, date_to=None):
             datetime.combine(fields.Date.from_string(date_from), time.min).replace(tzinfo=UTC),
             datetime.combine(fields.Date.from_string(date_to), time.max).replace(tzinfo=UTC)
         )
+
+    def redirect_to_project_task_form(self):
+        return {
+            'type': 'ir.actions.act_url',
+            'url': '/web#model=project.task&id=%s&action=%s&view_type=form' % (self.id, self.env.ref('project.action_view_my_task').id),
+            'target': 'new',
+        }
```

Comment:
```suggestion
        return {
            **self.env['ir.actions.act_window']._for_xml_id('project.action_view_my_task'),
            'res_id': self.id,
            'view_mode': 'form',
            'views': [[False, 'form']],
        }
```
This should work, right?

Reply:
![image](https://user-images.githubusercontent.com/64937092/232081837-e17c8f6a-f8c7-488a-8b28-5617c11beb4c.png)
that's what i got by testing your code

Reply:
Ok, maybe the entirety of my proposition isn't correct, but you get the idea right?

Reply:
You can let as you did, I did not catch it was for project sharing when we discuss about that.

The suggestion from @auon-odoo works if we already are in backend. 

Sorry about that.

Reply:
hey, okay thank you